configuration/messageproperties/messages_es.properties

```diff
@@ -50,7 +50,7 @@ pihcore.patientRegistration.contact.address.shortcut=(Optional) Quick search by
 
 pihcore.medicalhistory.hypertension=Hipertensión
 pihcore.medicalhistory.sti=ETS
-pihcore.acuteRheumaticFever.abbrev=ARF
```

Comment:
What's up with this change?  This doesn't look like an abbreviation.  Otherwise PR looks good.

Reply:
I logged into Transifex and confirmed that this was a change that Miguel made through the Editor (and just got pulled in to the message.properties files when I added this code), so I am going to assume this change was correct and intentional.  fyi @brandones